src/components/SearchList.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Resource } from '../store/resourcesSlice';
+import PreviewCard from './PreviewCard';
+import { RootState } from '../store/reducers';
+
+function fuzzysearch(needle, haystack) {
```

Comment:
It seems like all `fuzzysearch` does is return the number of characters in `haystack` that match `needle` in order, so a search term like "eeeee" would have the same score on two different articles named "eeeee" and "Hello, we need bread", even though it's unclear how the 2nd one would be relevant. Is there a reason we're using this method of matching? If not, I think it would make more sense to use a package like [Fuse](https://fusejs.io/), which does string matching using Levenshtein distance (edit distance from 170!). That way we'd only return exact matches or slight typos, which is a lot more intuitive than matching individual letters.

_(as a side note, i can't believe this [fuzzysearch code](https://github.com/bevacqua/fuzzysearch/pulls) got 2.5k stars on Github??!!)_

Reply:
i couldn't get fuse to deal w/ how we handle id's gonna rework that